frontend/lib/src/components/elements/Spinner/styled-components.ts

```diff
@@ -59,6 +59,11 @@ export const StyledSpinner = styled.div<StyledSpinnerProps>(
   })
 )
 
+export const StyledSpinnerTimer = styled.div(({ theme }) => ({
+  opacity: 0.6,
```

Comment:
This is using the same opacity as `st.caption`. Maybe it might make sense to turn that into a theming variable?